code/src/components/List.js

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { useSelector } from "react-redux"
+import styled from "styled-components"
+
+//--- component ---
+import { SingleTodo } from "./SingleTodo"
+
+export const List = () => {
+
+  //fetch all tasks from the store
+  const tasks = useSelector(store => store.todos.allTasks)
+
+  return (
+    <Todos>
+      {tasks.map((task) => (
+        <SingleTodo
+          key={task.id}
+          id={task.id}
+          text={task.text}
+          complete={task.complete}
+        />
+      ))}
+    </Todos>
+  )
+}
+
```

Comment:
Maybe set a limit to the characters typed, or style the task so it don't go outside the container when the todo-text gets to long?

Reply:
I did use `word-wrap: break-word; in the `TextContainer` styled component in `SingleTodo.js` which should do that job. Is the text still overflowing despite that?

Reply:
Yes, I took a picture, I'll send it to you on slack right away

Reply:
I have replaced it with `overflow-wrap: anywhere;` now.